docs/hive-integration/crds/agentClusterInstall-dualstack-SNO.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: extensions.hive.openshift.io/v1beta1
+kind: AgentClusterInstall
+metadata:
+  name: test-agent-cluster-install
+  namespace: spoke-cluster
+spec:
+  clusterDeploymentRef:
+    name: single-node
+  imageSetRef:
+    name: openshift-v4.8.0
+  networking:
+    clusterNetwork:
+    - cidr: 10.128.0.0/14
+      hostPrefix: 23
+    - cidr: fd01::/48
+      hostPrefix: 64
```

Comment:
So for dual stack clearly we will need to define two cidr blocks... but it isn't clear which cidr block is used for what. Another way to think about that question is what happens if I put two IP4 or two IP6 cidr blocks here? How does this differ from that?

Reply:
Right, so I think it still revolves around the fact that we need to have a place where we put what's supported and so on... To answer specifically your questions
1) Your pods will be getting IP address from this block(s). Depending on your manifest, your pods will be IPv4 or dual-stack, that's why you have 2 of them, but they are both citizens with the same rights
2) If you put two IPv4 and/or 2 IPv6 blocks, we will fail to deploy your cluster (via the validator) because this is not a legal combination from the perspective of the OCP (or maybe even Kubernetes). 

I think in principle, given that we are starting to do more and more advanced networking, it would be good if we had a Networking Intro document, but then again, question is "where?"